htdocs/css/main.css

```diff
@@ -374,6 +374,13 @@ table.grid tr.no_results td {
     text-align: center;
     font-style: italic;
 }
+/* if issue description is too wide */
+#issue_description {
+    display: inline-block;
+/* 82% from viewport width */
+    max-width: 82vw;
```

Comment:
According to:
https://css-tricks.com/viewport-sized-typography/ and
http://caniuse.com/#feat=viewport-units

IE 10+, Firefox 19+, Chrome 34+, Safari 7+, Android 4.4+, iOS 6+ - Supported

For IE should probably add some IE specific CSS hack, something like:
max-width: 995px\9;

Elan Ruusamäe wrote on 15.07.2015 16:59:
>
> In htdocs/css/main.css <https://github.com/eventum/eventum/pull/64#discussion_r34680474>:
>
> > @@ -374,6 +374,13 @@ table.grid tr.no_results td {
> >      text-align: center;
> >      font-style: italic;
> >  }
> > +/* if issue description is too wide */
> > +#issue_description {
> > +    display: inline-block;
> > +/* 82% from viewport width */
> > +    max-width: 82vw;
>
> i wonder what's browser compatibility for this css unit?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/eventum/eventum/pull/64/files#r34680474>.
>